packages/sync-actions/test/stores-sync.spec.js

```diff
@@ -0,0 +1,51 @@
+import storesSyncFn, { actionGroups } from '../src/stores'
+import { baseActionsList } from '../src/stores-actions'
+
+describe('Exports', () => {
+  test('action group list', () => {
+    expect(actionGroups).toEqual(['base'])
+  })
+
+  test('correctly define base actions list', () => {
+    expect(baseActionsList).toEqual([
+      { action: 'setLanguages', key: 'languages' },
+      { action: 'setName', key: 'name' },
+    ])
+  })
+})
+
+describe('Actions', () => {
+  let storesSync
+  beforeEach(() => {
+    storesSync = storesSyncFn()
+  })
+
+  test('should build `setName` action', () => {
+    const before = {
+      name: { locale: 'en', value: 'Algeria' },
+    }
+    const now = {
+      name: [
```

Comment:
The API accepts both `name` as an array and as a single value of `{ locale, value }`.
We would want to make sure in the MC that we pass in always `array`.

Because the end user might have the intention to update only the `en` value and retain `de`. If we pass in only `{ locale, value }` (not an array), we would end up overriding the entire `nameAllLocales`

Reply:
Really? Why isn't it documented then?

Reply:
one assumption is that maybe the  REST api is different from the Graphql api? 🤔 at least in graphql that is the case.